Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -118,6 +120,21 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddAuthentication(EdFiAuthenticationTypes.OAuth)
                 .AddScheme<AuthenticationSchemeOptions, EdFiOAuthAuthenticationHandler>(EdFiAuthenticationTypes.OAuth, null);
+
+            if (ApiSettings.IsFeatureEnabled(ApiFeature.IdentityManagement.GetConfigKeyName()))
+            {
+                services.AddAuthorization(config =>
+                {
+                    config.AddPolicy("UserPolicy", policyBuilder =>
+                    {
+                        policyBuilder.UserRequireCustomClaim(ClaimTypes.Email);
```

Comment:
This claim type seems wrong considering we are looking for a specific value that is not an email

Reply:
have created a sample Email claim, need to have a discussion on the same